website/.husky/pre-commit

```diff
@@ -0,0 +1,3 @@
+cd website
+yarn fmt && git add .
```

Comment:
can we use https://github.com/lint-staged/lint-staged to format only the git staged files? sometimes we don't `git add` all the files

Reply:
something like 
```sh
cd website
yarn lint-staged
```

and in the `package.json`, you can add:

```json
{
"scripts": {
...
"prepare": "husky install",
....
},
"lint-staged": {
    "*": [
      "prettier --write"
    ]
  },
}
```